src/style/blocks/cards.scss

```diff
@@ -0,0 +1,78 @@
+@import "../utils/variables";
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  width: 200px;
+  height: 406px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 0 16px;
+
+  &__image {
+    margin: 32px auto 40px;
+    width: 160px;
+    height: 134px;
+    background-image: url("images/imac.jpeg");
+    background-position: center;
+    background-size: 160px 134px;
```

Comment:
please add image by using `<img>` tag instead of background image and add descriptive alt attributte as is required due to the the checklist